docs/transport/collisions.rst

```diff
@@ -45,3 +46,18 @@ Spitzer_resistivity
 mobility
 Knudsen_number
 coupling_parameter
+
+Reference/API
+=============
+
+.. automodapi:: plasmapy.physics.transport.collisions
+      :skip: Debye_length
```

Comment:
`:skip:` is here while we haven't put `__all__` into every `.py` file (see #397) , so it'll stay here for a while, I suppose.